README.md

```diff
@@ -1 +1,33 @@
-# hello--microverse
\ No newline at end of file
+
+# hello-microverse
+
+> this the html and css linters project.
+
+
+## Built With
+
+- html
+- css
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+### Setup 
+clone the project or download the files to ypour local environment.
+
+### Install
+you have run the installation commands.
+
+### Usage 
+use linters to check the errors in your html and style 
+
+
+
+
+
+## Authors
+
```

Comment:
- [ ] consider adding more description to your readme. like social media links, project screenshots. Consider using the [Microverse readme template](https://github.com/microverseinc/readme-template).